tests/http/simple-spin-rust/Cargo.lock

```diff
@@ -148,7 +148,7 @@ dependencies = [
 
 [[package]]
 name = "spin-sdk"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
Is this intentional (in this PR?)

Reply:
Seems to be a side effect of building.  Perhaps someone made an earlier change but didn't check in Cargo.lock?

I can remove that from the commit and force-push, but I'm guessing it will pop up the next time anyone builds.

Reply:
Yeah seems fine to keep it, just making sure it wasn't accidentally pulled in from another change.